README.md

```diff
@@ -98,10 +98,46 @@ A problem occurred while evaluating entry:
 
 #### Properties inheritance
 It might be useful to have properties files that can recursively include
-other properties files (specified via an `include` property).
+other properties files (specified via an `include` property with including file path as value).
```

Comment:
I think the example you've given is really clear and this addition here will not be necessary.